macaroonbakery/tests/test_macaroon.py

```diff
@@ -1,14 +1,14 @@
 # Copyright 2017 Canonical Ltd.
 # Licensed under the LGPLv3, see LICENCE file for details.
-
 from unittest import TestCase
 
 import nacl.utils
 
 from macaroonbakery import LATEST_BAKERY_VERSION, BAKERY_V1, macaroon, codec
-from macaroonbakery.bakery import ThirdPartyInfo
 from macaroonbakery.macaroon import ThirdPartyLocator
-from macaroonbakery.third_party import legacy_namespace, ThirdPartyCaveatInfo
+from macaroonbakery.third_party import (
```

Comment:
It looks to me as if the test_add_third_party_caveat method in this file still needs to be updated, or is that going to be a separate PR?

Reply:
yep, next PR